src/components/Recipe/Recipe.js

```diff
@@ -0,0 +1,214 @@
+import axios from "axios";
+import { useState, useContext, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import AppContext from "../../context/AppContext";
+import {
+  Box,
+  Button,
+  Grid,
+  Typography,
+  Paper,
+  Table,
+  TableBody,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TableCell,
+} from "@mui/material";
+import RestaurantIcon from "@mui/icons-material/Restaurant";
+
+const Recipe = () => {
+  const [recipe, setRecipe] = useState({});
+  const [isRecipeOpen, setIsRecipeOpen] = useState();
```

Comment:
If this value is supposed to be a boolean - let's make it always a boolean!

```suggestion
  const [isRecipeOpen, setIsRecipeOpen] = useState(false);
```

Reply:
actually this value is never used! Let's remove it